Sources/Sentry/SentryFramesTracker.m

```diff
@@ -142,41 +152,44 @@ - (void)displayLinkCallback
     BOOL shouldRecordFrameRates = [SentryProfiler isRunning];
 #        endif // defined(TEST) || defined(TESTCI)
     BOOL hasNoFrameRatesYet = self.frameRateTimestamps.count == 0;
-    BOOL frameRateSignificantlyChanged
-        = fabs(self.frameRateTimestamps.lastObject[@"frame_rate"].doubleValue
-              - actualFramesPerSecond)
-        > 1e-10f; // these may be a small fraction off of a whole number of frames per second, so
-                  // allow some small epsilon difference
+    double previousFrameRate
+        = self.frameRateTimestamps.lastObject[@"frame_rate"].unsignedLongLongValue;
```

Comment:
`m`: `unsignedLongLongValue` and `double` don't match. I think it should be
```suggestion
    uint64_t previousFrameRate
        = self.frameRateTimestamps.lastObject[@"frame_rate"].unsignedLongLongValue;
```

Reply:
Excellent catch. I read the type keyword as "const auto" (got too spoiled by ObjC++ 😆 )